### DISCUSSION TITLE
What (if any) impact does the re-license of HashiCorp Vault to Business-Source-License (BSL) have on bank-vaults?

### DISCUSSION THREAD

**Discussion by JonTheNiceGuy** [bank-vaults/bank-vaults#2141] [Q&A] [answered]
Hi,

I was wondering whether this organisation has a view on HashiCorp's relicense of Vault under BSL? The relicense of Terraform has provoked opentf.org and I was wondering whether this causes any issues or impact to bank-vaults?

Thanks!

**Comment by sagikazarmark** [comment, marked as answer]
We are going to publish an initial statement soon.

**Reply by sagikazarmark** [reply]
Please see our preliminary announcement: https://github.com/orgs/bank-vaults/discussions/2146
